### ISSUE TITLE
boot can not work with Zulu JDK

### ISSUE THREAD

**Issue by dingmaotu** [boot-clj/boot#602]
Zulu OpenJDK distribution is not recognized by boot.exe. Only Oracle JDK is supported.

**Comment by Deraen** [comment]
This needs more information. Boot.exe is built with http://launch4j.sourceforge.net and it should find any JRE/JDK.

Is the JDK Available in %PATH%? (I'm not sure if that is used to find java in windows)

**Comment by dingmaotu** [comment]
@Deraen Yes it is in the path. I install Java 8 from http://www.azul.com/downloads/zulu/zulu-windows/, and confirmed that I can run java -version. However, boot.exe said it needs java 7 or later. Then I installed the official Oracle JDK, and it works fine. Based on this, I think launch4j uses some registry keys to find Java.

**Comment by dingmaotu** [comment]
Confirmed that this is an upstream problem: https://sourceforge.net/p/launch4j/feature-requests/103/
And this discussion provides a solution to this problem: https://github.com/kaikramer/keystore-explorer/issues/52.

**Comment by martinklepsch** [comment]
@dingmaotu Do you have any suggestions how we should proceed with this? I assume we are mostly dependent on Launch4j here?

**Comment by dingmaotu** [comment]
@martinklepsch Yes, and Launch4j is not interested to solve this problem in the near future. How about forking launch4j and making the change ourselves?

**Comment by martinklepsch** [comment]
Hm, I'm not too familiar with launch4j but forking it and maintaining such fork doesn't seem very attractive at this point. I remember seeing another issue/discussion about replacing launch4j with something different but can't find it right now. 

@dingmaotu maybe you can try improving launch4j and getting your change merged in? It seems so far that only an issue has been opened with noone stepping up to do the work.
